website/docs/d/nas_all_access_rules.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Network Attached Storage (NAS)"
+layout: "alicloud"
+page_title: "Alicloud: alicloud_nas_access_rules"
+sidebar_current: "docs-alicloud-datasource-nas-access-rules"
+description: |-
+  Provides a list of AccessRules owned by an Alibaba Cloud account.
+---
+
+# alicloud\_nas_access_rules
+
+This data source provides AccessRule available to the user.
+
+-> **NOTE**: Available in v1.189.0+.
+
+## Example Usage
+
+```terraform
+data "alicloud_all_nas_access_rules" "foo" {
+  access_group_name = "tf-testAccAccessGroupsdatasource"
+  source_cidr_ip    = "168.1.1.0/16"
+  rw_access         = "RDWR"
+  user_access       = "no_squash"
+  file_system_type  = "extreme"
+}
+
+output "alicloud_all_nas_access_rules_id" {
+  value = "${data.alicloud_nas_access_rules.foo.rules.0.id}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `access_group_name` - (Required ForceNew) Filter results by a specific AccessGroupName.
+* `ids` - (Optional, Available in 1.53.0+) A list of rule IDs.
+* `source_cidr_ip` - (Optional) Filter results by a specific SourceCidrIp. 
+* `user_access` - (Optional) Filter results by a specific UserAccess. 
+* `rw_access` - (Optional) Filter results by a specific RWAccess. 
+* `file_system_type` - (Required,ForceNew) The type of the file system: `standard `, `extreme`.
+* `ipv6_source_cidr_ip` - (Optional) The IPv6 address or CIDR block of the authorized object.
+                                                            You must set this parameter to an IPv6 address or CIDR block.
```

Comment:
This parameter cannot be set at the same time as the `source_cidr_ip ` parameter.